frontend/src/Components/HomePage/HomePage.js

```diff
@@ -0,0 +1,191 @@
+import { P, SIZES, WEIGHTS } from 'styles/StyledComponents/Typography';
+import DepartmentSearch from 'Components/Elements/DepartmentSearch';
+import { ICONS } from 'img/icons/Icon';
+import FjButton from 'Components/Elements/Button';
+import CardSkeleton from 'Components/Elements/Skeletons/CardSkeleton';
+
+function HomePage() {
+  const [{ data, loading, errors }, dispatch] = React.useReducer(fetchReducer, initialState);
+  const history = useHistory();
+  const theme = useTheme();
+
+  useEffect(() => {
+    async function _fetchMetaData() {
+      dispatch({ type: FETCH_START });
+      try {
+        const { data } = await axios.get(STATE_FACTS_URL);
+        const ncData = data.find((s) => s.state_key === 'nc');
+        dispatch({ type: FETCH_SUCCESS, payload: ncData });
+      } catch (e) {
+        dispatch({ type: FETCH_FAILURE, payload: e });
+        console.warn('error: ', e);
+      }
+    }
+    _fetchMetaData();
+  }, []);
+
+  return (
+    <ResponsiveInnerPage>
+      <S.HomePage>
+        <S.Heading>
+          <S.Logo white />
+          <S.SubTitle>
+            a project of{' '}
+            <S.SubTitleLink
+              href="https://www.forwardjustice.org"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              Forward Justice
+            </S.SubTitleLink>
+          </S.SubTitle>
+        </S.Heading>
+
+        <S.MainContent>
+          <S.About>
+            <S.AboutImage src={SplashImage} />
+            <S.AboutContent>
+              <S.AboutHeading>About the Data</S.AboutHeading>
+              {loading && <CardSkeleton />}
+              {data && (
+                <>
+                  <S.AboutSubSection>
+                    <S.AboutSubTitle size={SIZES[0]}>showing</S.AboutSubTitle>
+                    <P>
+                      <S.Datum>{Number(data.total_stops).toLocaleString()}</S.Datum> stops
+                    </P>
+                    <P>
+                      <S.Datum>{Number(data.total_searches).toLocaleString()}</S.Datum> searches
+                    </P>
+                  </S.AboutSubSection>
+                  <S.AboutSubSection>
+                    <S.AboutSubTitle>across</S.AboutSubTitle>
+                    <P>
+                      <S.Datum>{Number(data.total_agencies).toLocaleString()}</S.Datum>
+                      police and sheriff's departments
+                    </P>
+                  </S.AboutSubSection>
+                  <S.AboutSubSection>
+                    <S.AboutSubTitle>from</S.AboutSubTitle>
+                    <P weight={WEIGHTS[1]}>
+                      {data.start_date} - {data.end_date}
+                    </P>
+                  </S.AboutSubSection>
+                  <S.AboutExtra>
+                    All data comes directly from{' '}
+                    <S.FormLink
+                      href="https://fbaum.unc.edu/TrafficStops/SBI-122-form.pdf"
```

Comment:
We probably should ask FJ, but I bet we want to link to the SBI site: https://www.ncsbi.gov/Services/SBI-Forms/SBITrafficStop__FILLABLE.aspx